docker-compose/7.1.N-docker-compose.yml

```diff
@@ -164,7 +164,7 @@ services:
       - alfresco
       - share
   sync-service:
-    image: quay.io/alfresco/service-sync:3.7.2
+    image: quay.io/alfresco/service-sync:3.9.2
```

Comment:
Replace ths SS version with 3.9.0 instead of 3.9.2

Reply:
version changed to 3.9.0.